### PR TITLE
431 hightech patents

### LINKED ISSUE #431: High tech patents
Add indicator for high tech patents (a category in Eurostat's patent tables)

### PR DESCRIPTION
**Pull request by Jack-0-0**
Added high tech patents into make_eurostat_ip.py and related csvs and yaml files.

Closes #431

### CODE REVIEW

File: ds/data/processed/eurostat/epo_hightech_patent_applications.nuts2.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  value:
+    data_type: int
+    id: epo_hightech_patent_applications
+    label: High tech patent applications
+  year:
+    data_type: int
+    label: Year
+source_name: Eurostat
+source_url: https://ec.europa.eu/eurostat
+warning: The regional breakdown of the EU Member States is based on the Nomenclature of Territorial Units for Statistics – NUTS 2010 level 3
```
Comment (mindrones) [inline ds/data/processed/eurostat/epo_hightech_patent_applications.nuts2.yaml]: Do we mean `level 2` here or `level 3`?

**Review by mindrones — COMMENTED** [review]
I left a question below. Also, we need to add a new section to `CHANGELOG.md` titled `# 0.0.18 (next release)` and add this change in the `Indicators` section.

File: ds/data/processed/eurostat/epo_hightech_patent_applications.nuts2.yaml (hunk 1 above)
Comment (Jack-0-0) [reply ds/data/processed/eurostat/epo_hightech_patent_applications.nuts2.yaml]: It should be `level 2`

**Review by mindrones — APPROVED** [review]
Good to merge! 👌

File: ds/beis_indicators/eurostat_ip/make_eurostat_ip.py [hunk 2]
```diff
@@ -6,65 +6,128 @@
...
+    nuts_table = pd.read_excel(f"{PROJECT_DIR}/data/aux/NUTS_{year}.xls")
+    for l in [2, 3]:
+        nuts_codes[year][l] = set(
+            nuts_table.loc[
+                (nuts_table["COUNTRY CODE"] == "UK") & (nuts_table["NUTS LEVEL"] == l)
+            ]["NUTS CODE"]
+        )
+
+
+def create_nuts_list_spec(name):
```
Comment (dede95) [inline ds/beis_indicators/eurostat_ip/make_eurostat_ip.py:47]: I think a better way to set the spec year is to use [this function](https://github.com/nestauk/beis-indicators/blob/dev/ds/beis_indicators/geo/nuts.py#L117) . Or were the NUTS spec years specified in the dataset?

Comment (Jack-0-0) [reply ds/beis_indicators/eurostat_ip/make_eurostat_ip.py:47]: I got the NUTS spec from make_eurostat_ip.py where it said patents should use 2010 and trademarks should use 2013.
Does that function also work for nuts3? It looks like it is for nuts2?

Comment (dede95) [reply ds/beis_indicators/eurostat_ip/make_eurostat_ip.py:47]: @Jack-0-0 Okay, if that's the case, I'll approve the PR! 

You're right, it's only for nuts2.

**Review by dede95 — APPROVED** [review]

**Pull request merged**